tests/NHSD.BuyingCatalogue.Identity.Api.IntegrationTests/Steps/UserSteps.cs

```diff
@@ -7,29 +18,135 @@ namespace NHSD.BuyingCatalogue.Identity.Api.IntegrationTests.Steps
     [Binding]
     internal sealed class UserSteps
     {
+        private const string OrganisationMapDictionary = "OrganisationMapDictionary";
+        private const string AccessTokenKey = "AccessToken";
```

Comment:
I've refactored the access token to now be in config, as its in multiple places, probably a good idea to do the same with the dictionary

Reply:
*After a conversation with @sam-roberts-bjss I'm making a constants class and going to reference them from that